src/researches/getReadingTime.js

```diff
@@ -0,0 +1,10 @@
+let wordCountInText = require( "./wordCountInText.js" );
+let imageCount = require( "./imageCountInText.js" );
+
+module.exports = function( paper ) {
+	let numberOfWords = wordCountInText( paper );
+	let numberOfImages = imageCount( paper );
+
+	return Math.ceil( ( numberOfWords / 200 ) + ( numberOfImages * 0.2 ) );
```

Comment:
A specification of how this calculation came to be referencing the technical choices would be nice.